blocks/promotion/promotion.js

```diff
@@ -109,6 +109,7 @@ async function buildClock(block) {
   clock.className = 'rolex-frame';
   clock.innerHTML = `<iframe
       id="rolexFrame${placeholders.rolexId}"
+      title="rolex promotion"
```

Comment:
I apologize as this one feels really nitpicky, but if this title appears on hover, I think the brand name should be capitalized.

Reply:
Makes sense! 👍 Addressed in [2bdb5b2](https://github.com/hlxsites/pga-presidents-cup/pull/3/commits/2bdb5b222e309755ce9afe2256beb9c9e66fb5ee)